_studio/shared/src/mfx_vpp_vaapi.cpp

```diff
@@ -1647,7 +1647,14 @@ mfxStatus VAAPIVideoProcessing::Execute_Composition_TiledVideoWall(mfxExecutePar
                                *outputSurface);
         MFX_CHECK_WITH_ASSERT(VA_STATUS_SUCCESS == vaSts, MFX_ERR_DEVICE_FAILED);
 
-        outputparam         = m_pipelineParam[0];
```

Comment:
This looks to be initial setup of outputparam. You have wiped it out. Are you sure that we will not get uninitialized values?  Why you did not simply add code afterwards?